### PR TITLE
Dlg yaml file change

### PR DESCRIPTION
**Pull request by Osier-Yi**
1. The original version of dlg cannot successfully reconstruct the training data because of the default dropout =0.5 in the convnet2. In the modified yaml file: scripts/attack_exp_scripts/privacy_attack/reconstruct_fedavg_opt_on_femnist.yaml, we set model.dropout=0.
2.  in the unitest of dlg, we also check whether the dlg reconstruction loss is less than the threshold.

### CODE REVIEW

**Review by xieyxclack — APPROVED** [review]
LGTM
